app/src/main/java/com/github/palFinderTeam/palfinder/profile/ProfileAdapter.kt

```diff
@@ -100,6 +105,31 @@ class ProfileAdapter(
         }
     }
 
+    /**
+     * fill the 3 image placeholder with achievements pictures
+     */
+    private fun bindImages(holder: ProfileAdapter.ViewHolder, position: Int) {
+        val achievements = currentDataSet[position].achievements()
+        when (achievements.size) {
```

Comment:
Isn't there a more elegant way to achieve what is done in the 'when' here?

Reply:
I have tried several things, including a subfunction, and I think I cannot do it in less lines than this